### ISSUE TITLE
0.10.0 install issue

### ISSUE THREAD

**Issue by e1ven** [dr-prodigy/python-holidays#276]
I suspect you're aware, but I'm seeing an install issue with the just released 0.10.0.
(My CI is running against `holidays>=0.9.5`)

```python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-smsefye4/holidays/setup.py'"'"'; __file__='"'"'/tmp/pip-install-smsefye4/holidays/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-smsefye4/holidays/pip-egg-info
         cwd: /tmp/pip-install-smsefye4/holidays/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-smsefye4/holidays/setup.py", line 20, in <module>
        with codecs.open('holidays/__init__.py', 'r', 'utf-8') as fd:
      File "[redacted path]/lib/python3.6/codecs.py", line 897, in open
        file = builtins.open(filename, mode, buffering)
    FileNotFoundError: [Errno 2] No such file or directory: 'holidays/__init__.py'
```

**Comment by dr-prodigy** [comment]
Yes, I noticed this, in fact I've already removed 0.10.0 version from PyPi.
I will fix it and release a new version on PyPi ASAP.
Thank you for pointing this out: I'll keep you posted.
Cheers

**Comment by dr-prodigy** [comment]
Fixed and released on v0.10.1.. thank you again!
